sample-api-calls/SEA Modernization Starter Kit Environment (Local).postman_environment.json

```diff
@@ -0,0 +1,24 @@
+{
+	"id": "e50e829a-4d3d-4a97-80c5-f7ecb1825fbf",
+	"name": "SEA Modernization Starter Kit",
+	"values": [
+		{
+			"key": "ApiBaseUrl",
+			"value": "https://api-stage.ed-fi.org/v5.2/api",
```

Comment:
@semalaiappan could you change this to https://localhost/WebApi

Reply:
Fixed it . Thanks